### DISCUSSION TITLE
Where to find the list of all NER, POS, TAG with its description in the doc ?

### DISCUSSION THREAD

**Discussion by egbakou** [explosion/spaCy#11784] [Help: Other Questions] [answered]
The link [https://spacy.io/api/annotation](https://spacy.io/api/annotation) where we can find this information is not working.

**Comment by rmitsch** [comment, marked as answer]
Hi @egbakou, 

see this [reference in the docs on linguistic features](https://spacy.io/usage/linguistic-features#entity-types) - entity tags are listed as "Label scheme" in the descriptions of the published models, e.g. https://spacy.io/models/en#en_core_web_sm.

You can use [`spacy.explain()`](https://spacy.io/api/top-level#spacy.explain) to obtain explanations for entity types.

**Reply by egbakou** [reply]
Thnak you @rmitsch
